.codecov.yml

```diff
@@ -7,7 +7,6 @@ coverage:
     project:
       default:
         # TODO(texasaggie97) - target lowered when adding system test coverage. Raise as coverage rises.
-        # TODO(texasaggie97) - update target when files other than _library.py removed from coverage report
```

Comment:
TODO is gone but target didn't change, what gives?

Reply:
Lost the change somehow. Put back.